examples/tutorials/02_cozmo_face/02_cozmo_face_mirror.py

```diff
@@ -92,25 +95,5 @@ def cozmo_face_mirror(robot):
         time.sleep(duration_s)
 
 
-def run(sdk_conn):
-    '''The run method runs once Cozmo is connected.'''
-
-    robot = sdk_conn.wait_for_robot()
-    get_in_position(robot)
-
-    print("Press CTRL-C to quit")
-
-    try:
-        cozmo_face_mirror(robot)
-    except KeyboardInterrupt:
-        print("")
-        print("Exit requested by user")
-
-
-if __name__ == '__main__':
-    cozmo.setup_basic_logging()
-    cozmo.robot.Robot.drive_off_charger_on_connect = False  # Cozmo can stay on his charger for this example
-    try:
-        cozmo.connect(run)
-    except cozmo.ConnectionError as e:
-        sys.exit("A connection error occurred: %s" % e)
\ No newline at end of file
+cozmo.robot.Robot.drive_off_charger_on_connect = False  # Cozmo can stay on his charger for this example
```

Comment:
I've added a multi-line explanation into 01_cozmo_face_image.py as that's the first place this is encountered. The docstrings for the action, property, etc. all describe some of this too.